next-sitemap.config.js

```diff
@@ -49,6 +49,10 @@ module.exports = {
         disallow: [
           "/404",
           "/500",
+          "/*Browse.aspx$",
```

Comment:
Please add the following too:
/* browsexml.aspx* (I intentionally added a space due to markdown display - remove the space when you implement it)
/*browsexml.aspx$